docs/hosting/installation/npm.md

```diff
@@ -77,6 +77,15 @@ Start n8n with `--tunnel` by running:
 n8n start --tunnel
 ```
 
+## Reverting an upgrade
+
+Install the older version that you want to go back to.
+
+If the upgrade involved a database migration:
+
+1. Check the feature documentation and release notes to see if there are any manual changes you need to make.
+1. Run `n8n db:revert` on your current version to roll back the database. If you want to revert more than one database migration, you need to repeat this process.
```

Comment:
Also worth noting that RBAC can only be reverted all created projects are deleted

Reply:
I'll include that in the release note. This is just generic info.